README.md

```diff
@@ -77,3 +77,22 @@ in an environment that has bioblend and Python 3. Then execute:
 ```
 run_flavour_workflows.sh
 ```
+
+# Setting up access to a Galaxy instance
+
+To run the Galaxy part, you will need a running Galaxy instance with all tools installed. Below we explain with [Human Cell Atlas use-galaxy.eu](https://humancellatlas.usegalaxy.eu/) as an example which already has the tools, but the same holds for another instance where the Galaxy tools are installed. 
+
+## Using Human Cell Atlas use-galaxy.eu instance
+
+The [Human Cell Atlas use-galaxy.eu](https://humancellatlas.usegalaxy.eu/) Galaxy instance already has all the tools required installed there, and can be used to reproduce the Expression Atlas clustering pipeline available here. For this you need to: 
+- Create an account at https://humancellatlas.usegalaxy.eu/ by clicking on **Login or Register**
+![image](https://user-images.githubusercontent.com/368478/62038201-35d49300-b1ed-11e9-9c87-571cf539cb8c.png)
+- Retrieve you users API Key for programmatic access:
```

Comment:
Typo: "you user" should be "your user's"

Reply:
Well spotted, thanks, will fix.